src/main/resources/admin.vm

```diff
@@ -16,6 +16,10 @@
         <label for="secret">$i18n.getText("lgtm-servlet.admin.secret.label")</label>
         <input type="text" id="secret" name="secret" class="text">
       </div>
+      <div class="field-group">
+        <label for="secret">$i18n.getText("lgtm-servlet.admin.externalHookUrl.label")</label>
+        <input type="text" id="externalHookUrl" name="externalHookUrl" class="externalHookUrl">
```

Comment:
This new field is being rendered strangely in my browser. Should this be `class="text"` like the other fields?

Reply:
In addition the `for` is wrong.